lib/databases/postgresql

```diff
@@ -112,6 +125,8 @@ EOF
         if is_fedora; then
             sudo systemctl enable postgresql
         fi
+    elif is_clearlinux; then
+        echo "On Clearlinux no need to install."
```

Comment:
This is only true with our image, what would an upstream Clear image require here?

Reply:
Ok, I will try on a normal clearlinux image and find out what bundle we need to put here.